ner_v2/detectors/temporal/date/en/date_detection.py

```diff
@@ -29,10 +27,15 @@ class DateDetector(object):
         tag: entity_name prepended and appended with '__'
         timezone: Optional, pytz.timezone object used for getting current time, default is pytz.timezone('UTC')
         now_date: datetime object holding timestamp while DateDetector instantiation
+<<<<<<< HEAD
+        day_dictionary: dictonary mapping day indexes to day of week spellings and 
```

Comment:
W291 trailing whitespace

Reply:
This is merge error in docstring, will fix it later